frontend/app/src/components/pincode/Pincode.svelte

```diff
@@ -0,0 +1,189 @@
+<script>
+    /**
```

Comment:
Pretty surprised these got through all the linting! 

Would be nice to convert these to typescript and scss.